clap.zig

```diff
@@ -2060,64 +2064,76 @@ pub fn usage(stream: anytype, comptime Id: type, params: []const Param(Id)) !voi
                 try cs.writeAll("...");
         }
 
-        try cs.writeByte(']');
+        try cs.writeAll("]");
+        try cs.flush();
     }
 
-    if (!has_positionals)
-        return;
-
-    for (params) |param| {
-        if (param.names.short != null or param.names.long != null)
-            continue;
+    if (has_positionals) {
+        for (params) |param| {
+            if (param.names.short != null or param.names.long != null)
+                continue;
 
-        if (cos.codepoints_written != 0)
-            try cs.writeAll(" ");
+            if (cos.codepoints_written != 0)
+                try cs.writeAll(" ");
 
-        try cs.writeAll("<");
-        try cs.writeAll(param.id.value());
-        try cs.writeAll(">");
-        if (param.takes_value == .many)
-            try cs.writeAll("...");
+            try cs.writeAll("<");
+            try cs.writeAll(param.id.value());
+            try cs.writeAll(">");
+            if (param.takes_value == .many)
+                try cs.writeAll("...");
+            try cs.flush();
+        }
     }
 }
 
 fn testUsage(expected: []const u8, params: []const Param(Help)) !void {
     var buf: [1024]u8 = undefined;
-    var fbs = std.io.fixedBufferStream(&buf);
-    try usage(fbs.writer(), Help, params);
-    try std.testing.expectEqualStrings(expected, fbs.getWritten());
+    var writer = std.io.Writer.fixed(&buf);
+    try usage(&writer, Help, params);
+    try std.testing.expectEqualStrings(expected, writer.buffered());
 }
 
-test "usage" {
-    @setEvalBranchQuota(100000);
+test {
```

Comment:
Hmm, can we leave out splitting this test case into multiple blocks for now? I can see how this is useful, but I'll need to ponder on it further